cl/people_db/fixtures/tenn_judges.json

```diff
@@ -0,0 +1,1494 @@
+[
+    {
+        "fields": {
+            "ftm_eid": null,
+            "name_first": "Allen",
+            "ftm_total_received": null,
+            "dod_city": "",
+            "dob_state": "",
+            "id": 9694,
```

Comment:
This `id` field gave me pause. We use fixtures to import new courts all the time, but we can do that because we give them explicit IDs. I don't like just giving these IDs this way and doing a it quite this way. I think we need to take a step back and come up with a different approach that doesn't rely on IDs that in turn rely on the state of our DB at this instant.

Want to take a look in the data migrations docs and see if there's a better approach there?

Reply:
yes, I would love to find a better solution - but im not sure what it is yet.

Reply:
I have a solution, but im not sure you are going to like it.

Reply:
Ominous!

Reply:
Removed migration in favor of loading it into the system directly and modified the fixture for testing purposes